test/Microsoft.Azure.Devices.Edge.Test.Common/linux/EdgeDaemon.cs

```diff
@@ -132,8 +130,7 @@ public Task ConfigureAsync(Func<DaemonConfiguration, Task<(string, object[])>> c
 
         async Task InternalStartAsync(CancellationToken token)
         {
-            string[] output = await Process.RunAsync("systemctl", "start iotedge", token);
-            Log.Verbose(string.Join("\n", output));
+            await Process.RunAsync("systemctl", "start iotedge", token);
```

Comment:
Hmm... This command _will_ be different for release/1.2 onward i.e. `iotedge` vs `aziot-edge`.

Reply:
Correct. That change is already in master.

https://github.com/Azure/iotedge/blob/master/test/Microsoft.Azure.Devices.Edge.Test.Common/linux/EdgeDaemon.cs#L131